pkg/helm/helm.go

```diff
@@ -0,0 +1,45 @@
+package helm
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// Uninstall uninstalls a helm chart of the given name. There is no error in the case
+// of the helm chart not existing.
+func Uninstall(chartName string) error {
+	helmArgs := []string{"uninstall", chartName}
+	cmd := exec.Command("helm", helmArgs...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		if strings.Contains(string(output), "not found") {
+			return nil
+		}
+		return fmt.Errorf("error executing command: %s %s", err, output)
+	}
+	return nil
```

Comment:
[q/nit] WDYT of extracting this and the one below to a function like

```golang
executeHelmCommand(args []string, allowedErrors string...) error {
	cmd := exec.Command("helm", helmArgs...)
	output, err := cmd.CombinedOutput()
	if err != nil {
		// check if the error contains one of the allowedErrors, return nil in that case
		return fmt.Errorf("error executing command: %s %s", err, output)
	}
	return nil
}
```

?

Reply:
yes makes sense!